### PR TITLE
Generate single test suite for mock and live testings

### PR DESCRIPTION
**Pull request by Nuvindu**
# Description

$subject

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -191,43 +191,31 @@ Execute the commands below to build from the source.
    ./gradlew clean test
    ```
 
-3. To run a group of tests
```
Comment (RDPerera) [inline README.md:194]: Was excluding the guide for running tests on environments intentional?

Comment (Nuvindu) [reply README.md:194]: thought of removing the guide since it is not necessary for this package as there are no test groups now

Comment (RDPerera) [reply README.md:194]: Yes, the guide for running tests on a different group should be removed. However, I'm asking about the guide for setting up the environments so users can test on live endpoints or mock servers (it's already available under tests/README.md ). Do we need to exclude it as well ?

Comment (Nuvindu) [reply README.md:194]: I have removed the details about test groups from the README.md file and included only instructions on setting up environmental variables to execute either live or mock tests.

**Review by RDPerera — APPROVED** [review]
LGTM

**Pull request merged**